code/script.js

```diff
@@ -222,104 +228,152 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+//FUNCTIONS FOR GLOBAL VARIABLES//
+// RANDOMLY SELECT A CHARACTER FROM ARRAY and set as the value of the variable called secret//
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(secret)
 }
 
-// This function to start (and restart) the game
+//START FUNCTION TO START (and restart) GAME//
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
+  //Link to all characthers from start
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  // Invokes the generateBoard function to display the board of charachters when website is loaded
+  generateBoard()
+  // Invokes the setSecret charachter function when website is loaded 
+  setSecret()
+  console.log("Start function called");
 }
 
-// setting the currentQuestion object when you select something in the dropdown
+//DROPDOWN MENU - QUESTIONS//
+// SETTING THE currentQuestion OBJECT when you select something in the dropdown menu
 const selectQuestion = () => {
+  console.log(selectQuestion)
```

Comment:
Hello Mirela, 

Nice Work Mirela! 

your code looks mostly fine, but there are a few minor issues and improvements that can be made:

In the selectQuestion function, you have a console.log statement with console.log(selectQuestion). You should log the actual values or the result of the function, like console.log(currentQuestion), to see the selected question.